backend/app/services/chat/preprocessing/contexts.py

```
@@ -0,0 +1,593 @@
+                        subtask_id=subtask_id,
+                        user_id=user_id,
+                        context_type=ContextType.KNOWLEDGE_BASE.value,
+                        name=kb_name,
+                        status=ContextStatus.READY.value,
+                        type_data={
+                            "knowledge_id": int(knowledge_id) if knowledge_id else 0,
+                            "document_count": document_count,
+                        },
+                    )
+                    kb_contexts_to_create.append(kb_context)
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)